extensions/yaml_support/labgraph_monitor/server/serializer_node.py

```diff
@@ -56,6 +56,10 @@ def add_message_1(self, message: RandomMessage) -> None:
             "grouping": grouping,
             "timestamp": message.timestamp,
             "numpy": list(message.data),
+
```

Comment:
Unnecessary space, have the black and flake8 checks been run? Reference: https://github.com/facebookresearch/labgraph/pull/76